lib/cc/engine/analyzers/php/visitor.rb

```diff
@@ -182,21 +182,17 @@ def accept(target)
 
           ALL_NODES.each do |type|
             eval <<-RUBY
-              def visit_#{type}Node(node)
-                node.sub_nodes.map do |sub_node|
+              def visit_#{type}Node(o)
```

Comment:
can you use something more descriptive than `o`?  (i know that's what was there before)